ghost/i18n/locales/pt-BR/newsletter.json

```diff
@@ -5,10 +5,10 @@
     "Email": "E-mail",
     "free": "grátis",
     "Keep reading": "Continue lendo",
-    "Less like this": "Não gostei",
+    "Less like this": "Não gostei, não mostrar mais como este",
     "Manage subscription": "Gerenciar assinatura",
     "Member since": "Membro desde",
-    "More like this": "Gostei",
+    "More like this": "Gostei, mostra mais com este",
```

Comment:
This is a label for the thumbs up or down icon in the newsletter. It needs to be short.

Reply:
This translation is not grammatically wrong, but it sounds weird. I would just leave "Gostei", without the rest.